otter/integration/tests/test_convergence.py

```diff
@@ -868,7 +872,7 @@ def test_servers_that_build_for_too_long_time_out_and_are_replaced(self):
             criteria=[{"server_name": server_name_prefix + ".*"}],
             behaviors=[
                 {"name": "build",
-                 "parameters": {"duration": otter_build_timeout * 2}},
+                 "parameters": {"duration": otter_build_timeout * 3000}},
```

Comment:
Note to reviewer - I saw that this test seems to intermittently fail sometimes on CI - am hoping this will be a quick fix (I didn't see any errors from the otter logs, so I figured things just got delayed so that the server became active) - just snuck it in here.